src/cheri_pte.sail

```diff
@@ -16,17 +16,27 @@ bitfield PTE_Bits : pteAttribs = {
 /*
  * Reserved PTE bits used by extensions on RV64.
  *
- * There are no such bits on RV32/SV32, so we hard-code both StoreCap=1
- * and LoadCap=1 (i.e., there is no ability to disable capability loads
+ * There are no such bits on RV32/SV32, so we hard-code both CapWrite=1
+ * and CapRead=1 (i.e., there is no ability to disable capability loads
  * from or stores to pages in the SV32 MMU).
  */
 type extPte = bits(10)
 
 bitfield Ext_PTE_Bits : extPte = {
-  StoreCap     : 9, /* Permit capability stores */
-  LoadCap      : 8, /* Permit capability loads */
+  CapWrite     : 9, /* Permit capability stores */
+  CapRead      : 8, /* Permit capability loads */
 }
 
+/*
+ * CapWrite     = 1,
+ * CapRead      = 1,
+ * CapDirty     = 1,
+ * LoadCap_Mod  = 0,
+ * LoadCap_Gen  = 0,
```

Comment:
These should be reserved 0 for now as you haven't actually implemented them here